documents/motions/16.2020.1 - Disputes.md

```diff
@@ -1,20 +1,20 @@
 # SUBMISSION OF PROPOSED MOTION
 
-**Motion number:** 16.2019.1  
+**Motion number:** 16.2020.1  
 **Subject:** Disputes  
 **Intent:** Definition of the Processes in case of Disputes  
 **Submitted by:** Board of Directors  
-**Date:** January 1, 2019  
+**Date:** September 30, 2020  
 
 # Motion
 
 1. All acts and decisions of WCA Staff may be subject to disputes by affected individuals.
    1. Disputes regarding the WCA Regulations must be directed to and handled by WCA Regulations Committee, and are bound by the provisions of the WCA Regulations.
       1. Appeals regarding the final decision must be directed to and handled by WCA Board of Directors.
-   2. Disputes regarding the WCA Bylaws and the Motions must be directed to and handled by WCA Board of Directors, and are bound by the provisions of the WCA Bylaws and the Motions and the applicable governing laws.
-      1. Appeals regarding the final decision must be directed to and handled by WCA Ethics Committee.
-   3. All other Disputes must be directed to and handled by WCA Disciplinary Committee, and are bound by the WCA Regulations, the WCA Bylaws and the Motions, and the applicable governing laws.
-      1. Appeals regarding the final decision must be directed to and handled by WCA Ethics Committee.
+   2. Disputes regarding the WCA Bylaws and the Motions should be directed to and handled by WCA Board of Directors, and are bound by the provisions of the WCA Bylaws and the Motions and the applicable governing laws.
```

Comment:
Any particular reason why this was changed? Are there situations in which these disputes shouldn't be handled by the party described in each section?

Reply:
Yes there are some cases where a conflict may exist for example if someone were to bring a dispute about how the Board handled the motions/Bylaws then it would be impractical for the Board to handle this case. Likewise generally if a member of the committee is involved in the dispute then it shouldn't be handled by that committee and it usually isn't. This change is mainly just to make the motions match what actually happens in practice

Reply:
Thanks!